test/tomo/backends/astra_setup_test.py

```diff
@@ -50,11 +50,11 @@ def _discrete_domain(ndim, interp):
     space : `DiscreteLp`
         Returns a `DiscreteLp` instance
     """
-    minpt = np.arange(-1, -ndim - 1, -1)  # -1, -2 [, -3]
-    maxpt = np.arange(1, ndim + 1, 1)  # 1, 2 [, 3]
+    min_pt = np.arange(-1, -ndim - 1, -1)  # -1, -2 [, -3]
+    max_pt = np.arange(1, ndim + 1, 1)  # 1, 2 [, 3]
```

Comment:
You felt the need to comment in order to clarify, which is a red flag. What about `max_pt = np.arange(1, ndim + 1)` and `min_pt = -max_pt`?